io.github.insilmaril.vym.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.insilmaril.vym
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: vym
+rename-desktop-file: vym.desktop
+rename-icon: vym #128x128
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --own-name=org.insilmaril.*
+
+modules:
+  - name: vym
+    buildsystem: cmake-ninja
+    post-install:
+      - install -Dm644 icons/vym-128x128.png /app/share/icons/hicolor/128x128/apps/vym.png
```

Comment:
this isn't installed by upstream? It's a bug and should be fixed upstream too.

Reply:
I mentioned it to the developer